app/objects/c_planner.py

```diff
@@ -10,10 +10,12 @@ def unique(self):
 
     @property
     def display(self):
-        return dict(name=self.name, module=self.module, params=self.params, description=self.description)
+        return dict(name=self.name, module=self.module, params=self.params, description=self.description,
+                    stopping_conditions=[fact.display for fact in self.stopping_conditions])
 
-    def __init__(self, name, module, params, stopping_conditions=None, description=None):
+    def __init__(self, id, name, module, params, stopping_conditions=None, description=None):
```

Comment:
send this parameter in as identifier instead of id, which is a reserved python keyword